sentry-spring-boot/src/main/java/io/sentry/spring/boot/SentryAutoConfiguration.java

```diff
@@ -298,7 +292,7 @@ public FilterRegistrationBean<SentryTracingFilter> sentryTracingFilter(
       @Configuration(proxyBeanMethods = false)
       @ConditionalOnClass(HandlerExceptionResolver.class)
       @Open
-      static class SentryExceptionResolverConfigurationWrapper {
+      static class SentryNonServletOnlyModeConfig {
```

Comment:
Just a suggestion, but wouldn't something like `SentryMvcModeConfig`, be more easily understandable?